substrate/frame/revive/fixtures/build.rs

```diff
@@ -127,6 +127,8 @@ fn invoke_build(current_dir: &Path) -> Result<()> {
 		.env("PATH", env::var("PATH").unwrap_or_default())
 		.env("CARGO_ENCODED_RUSTFLAGS", encoded_rustflags)
 		.env("RUSTUP_HOME", env::var("RUSTUP_HOME").unwrap_or_default())
+		// Support compilation on stable rust
+		.env("RUSTC_BOOTSTRAP", "1")
```

Comment:
@athei looked like it was set before and removed in https://github.com/paritytech/polkadot-sdk/pull/6565/files#diff-62eca32e47a16e20f6e87347c342f29a44cffd066f00ad412672bd0a7f8399da

can't remember why

Reply:
It was removed because we were added a `rust-toolchain.toml` file so that a specific nightly is always used. There was no stable version available at  that time that could compile for riscv64e. So it made sense to remove it.

Since Rust `1.84` stable can compile this. So I suggest we remove the toolchain file as soon as the CI is updated to the newest stable.